admin/files/online-prezence/online-prezence-potvrzovaci-modal.js

```diff
@@ -0,0 +1,34 @@
+/* logika k situaci, že není zaškrtnutý ani jeden účastník k aktivitě  */
+function zadnyUcastnikNeniPotvrzen() {
+  const checkboxy = document.querySelectorAll(`#ucastniciAktivity${posledneKliknutaAktivitaId} input.dorazil[checked]`)
+  return checkboxy.length === 0
```

Comment:
Navrhuju zkrátit původní funkci na tohle (využít naplno `querySelectorAll`)

Původní kód ani nefungoval - nepoznalo to zaškrtnuté účastníky, ale nepřišel jsem na to proč. Přišel mi v pořádku.